# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../torcs/gcc41.patch
# Copyright (C) 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

diff -Nur torcs-1.2.4-orig/src/modules/graphic/ssggraph/grscreen.h torcs-1.2.4/src/modules/graphic/ssggraph/grscreen.h
--- torcs-1.2.4-orig/src/modules/graphic/ssggraph/grscreen.h	2005-08-24 19:19:47.000000000 +0000
+++ torcs-1.2.4/src/modules/graphic/ssggraph/grscreen.h	2006-04-04 22:07:12.686417000 +0000
@@ -78,7 +78,7 @@
 
     void update(tSituation *s, float Fps);
 
-    void cGrScreen::camDraw(tSituation *s);
+    void camDraw(tSituation *s);
 
     void updateCurrent(tSituation *s);
 
